README.md

```diff
@@ -51,14 +39,53 @@ Simply run the command bellow to build the frontend with React.
 yarn build
 ```
 
-A new `build` directory has now appeared, it's an static export of the react app, you can place it whereever you want.
+A new `build` directory has now appeared, it's an static export of the react app, you can place it anywhere you want, e.g. /path/to/vault-build.
+
+## Nginx configuration
+
+You should not expose your backend URL, so you can easily create a nginx vhost configuration to serve both static files and proxy to the backend :
+
+```nginx
+server {
+	listen 443 ssl http2;
+	listen [::]:443 ssl http2;
+	server_name vault.your-company.fr;
+	
+	ssl_certificate ...;
+	ssl_certificate_key ...;
+	... other ssl stuff ...;
+
+	root /path/to/vault-build;
+	index index.html;
+	location / {
+		try_files $uri $uri/ =404;
+	}
+	location /backend/ {
+		proxy_pass http://127.0.0.1:1337/;
+	}
+}
+```
+
 
 ## Build and install yopass-server
+This repo is not design to provide support on yopass solution, this is just given as example and reminder
+(on Debian 11)
 ```bash
 sudo apt-get install golang
+cd ~
 git clone https://github.com/jhaals/yopass.git
 cd yopass
 go build ./cmd/yopass-server
+sudo mv yopass-server /usr/local/bin/
```

Comment:
Maybe we should add instruction for the chmod thing ?

Reply:
yopass-server binary is already executable when compil is finished